src/pages/comunidad/index.tsx

```diff
@@ -1,18 +1,50 @@
+import { GetStaticProps } from 'next';
+import { useRouter } from 'next/router';
 import Link from 'next/link';
 import Layout from '../../components/Layout';
 import ProfileCard from '../../components/ProfileCard';
-import { GetStaticProps } from 'next';
 
 import { Profile } from '../../lib/types';
 import { getLayout } from '@/utils/get-layout';
 import { getProfiles } from '../api/google-sheet';
+import { useCallback, useEffect, useState } from 'react';
+import { motion } from 'framer-motion';
 
 type PostsPageProps = {
   profiles: Profile[];
   preview?: boolean;
 };
 
 const ProfilesPage: React.FC<PostsPageProps> = ({ profiles, preview }) => {
+  const router = useRouter();
+  const [filter, setFilter] = useState(null);
+
+  // Non repeated available technologies.
+  const technologies = Array.from(
+    new Set(profiles.map((profile) => profile.technologies).flat()),
```

Comment:
TS gave me problems using [... new Set ([1, 2, 2])] to avoid repeated technologies.